csrc/scheduler/mma_utils.cpp

```diff
@@ -703,11 +704,51 @@ std::unordered_set<IterDomain*> getMmaDomainSet(
   return {mma_domains.begin(), mma_domains.end()};
 }
 
+bool isConsumerAllocationInnerIDProducerAllocationInnerID(
+    TensorView* consumer) {
+  // We are only looking at the allocation domains accross
+  // the producer and consumer of a LoadStoreOp
+  auto LdStOp = consumer->definition();
```

Comment:
You can use `consumer->definition()-as<LoadStoreOp>()`. Also nit: we usually use lower snake case for variable names.

Reply:
I'll remove the requirement for a LoadStoreOp here.